dev-modules/babel-plugin-inline-webgl-constants/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "babel-plugin-inline-webgl-constants",
-  "private": true,
-  "version": "9.0.0-alpha.47",
+  "version": "2.0.0-alpha.1",
```

Comment:
I think we can just publish 2.0.0, 2.01, etc.

Reply:
Sounds good! I've used the version strings that are already published but it could make sense to do a stable publish for each, either here or after moving into a separate repo if we prefer.